ui/src/app/app.tsx

```diff
@@ -124,24 +122,69 @@ export class App extends React.Component<{}, {version?: Version; popupProps: Pop
             <Provider value={providerContext}>
                 {this.state.popupProps && <Popup {...this.state.popupProps} />}
                 <Router history={history}>
-                    <Switch>
-                        <Redirect exact={true} path={uiUrl('')} to={workflowsUrl} />
-                        <Redirect from={timelineUrl} to={uiUrl('workflows')} />
+                    <Layout navItems={navItems} version={() => <>{this.state.version ? this.state.version.version : 'unknown'}</>}>
+                        <Notifications notifications={this.notificationsManager.notifications} />
+
                         <ErrorBoundary>
```

Comment:
move this inside the layout, so we can keep button even in case of a "ops!" error